src/QMCWaveFunctions/EinsplineAdoptor.h

```diff
@@ -248,6 +248,11 @@ struct BsplineSet: public SPOSetBase, public SplineAdoptor
     SplineAdoptor::set_spline(spline_r,spline_i,twist,ispline,level);
   }
 
+  inline RealType RATIO(const ParticleSet& P, int iat, const ValueType* restrict arow)
```

Comment:
A method name that is all capital letters seems confusing because it's normally used for pre-processor defines and constants.